<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesShutdown</class>
 <widget class="QWidget" name="PreferencesShutdown">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>310</height>
   </rect>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="PowerManagemetGroupBox">
     <property name="title">
      <string>Power Management</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="shutdownMethodLabel">
        <property name="toolTip">
         <string>Select system shutdown mode</string>
        </property>
        <property name="text">
         <string>System shutdown mode:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="kcfg_shutdownMethods"/>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="actionGroupBox">
     <property name="title">
      <string>Scheduler</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="kcfg_timerRadioButton">
          <property name="text">
           <string>in:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="kcfg_scheduleDateTime">
          <property name="displayFormat">
           <string>HH'h'mm'm'</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="kcfg_jobsRadioButton">
        <property name="text">
         <string>when all jobs are finished</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="advancedGroupBox">
     <property name="title">
      <string>Advanced</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="kcfg_pausedShutdown">
        <property name="toolTip">
         <string>Do not shutdown the system if some files are set on pause</string>
        </property>
        <property name="text">
         <string>if jobs are finished but paused files remain</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>350</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
